.github/workflows/.nvmrc

```diff
@@ -0,0 +1 @@
+echo "16.16.0" > .nvmrc
```

Comment:
EOF

Reply:
Also, 
```suggestion
16.16.0
```
And please put the file on the root of the project

Reply:
@danielgalvis98 changes made